### ISSUE TITLE
Update our protocol version handshake message

### ISSUE THREAD

**Issue by pospeselr** [blueprint-freespeech/ricochet-refresh#36]
Currently using whatever the original ricochet version string, which is hella wrong now.

**Comment by m-simonelli** [comment]
I proposed a fix when working on file_transfer in a comment, I'll copy it here:
lines are based on m-simonelli/file_transfer

libtego/source/protocol/Connection_p.h (line 47):
```
    static const quint8 ProtocolVersion = 1;
```
should become
```
    static const quint8 ProtocolVersion = 2; // or should it be 3 since we left ricochet v2 and skipped to ricochet v3?
```

libtego/source/protocol/Connection.cpp (line 189):
```
    char intro[] = { 0x49, 0x4D, 0x02, ProtocolVersion, 0 };
```
should become
```
    char intro[] = { 0x49, 0x4D, 0x01, ProtocolVersion };
```

**Comment by m-simonelli** [comment]
Solved by #39
